README.md

```diff
@@ -84,3 +84,7 @@ steps:
 ## 🤝 Contributions
 
 Contributions are welcome! :heart: Please read our [Contributing Guide](.github/CONTRIBUTING.md) for information about how to get involved in this project.
+
+<!--
```

Comment:
This is an HTML comment so nobody will see it when they view the readme via github. Is that intentional?